test/frame_test.dart

```diff
@@ -236,6 +236,52 @@ void main() {
   });
 
   group('.parseFirefox/.parseSafari', () {
+    test('parses a Firefox stack trace with anonymouse function correctly', () {
```

Comment:
[nit] typo in anonymous. Also could shorten a bit since "correctly" is implied.

```suggestion
    test('parses a Firefox stack trace with anonymous function', () {
```

Reply:
fixed, thank you!